src/elona/ui/ui_menu_mod_info.cpp

```diff
@@ -0,0 +1,263 @@
+#include "ui_menu_mod_info.hpp"
+
+#include "../../util/fileutil.hpp"
+#include "../../util/strutil.hpp"
+#include "../audio.hpp"
+#include "../draw.hpp"
+#include "../i18n.hpp"
+#include "../lua_env/mod_manager.hpp"
+
+
+
+namespace elona
+{
+namespace ui
+{
+
+static int _wrap_text(std::string& text, int max_line_length)
```

Comment:
This function will be reused in other places and isn't strongly tied to the menu, so it should be moved to `text.cpp` or `util/strutil.hpp`.